index.bs

```diff
@@ -10552,29 +10552,27 @@ of <a href="#rendering-a-graph">rendering a graph</a>.
 
 The system level audio callback is also queued as a task in the
 <a href="#control-message-queue">control message queue</a>. The UA MUST perform
-the following sequence to process render quanta to fulfill such task by filling
-up the requested buffer size. Along with the primary message queue, the
+the following algorithms to process render quanta to fulfill such task by
+filling up the requested buffer size. Along with the primary message queue, the
 <a>rendering thread</a> has another task queue for microtasks for any microtask
 operation such as resolution of {{Promise}}s in the {{AudioWorkletGlobalScope}}.
 
-<div id="rendering-a-graph" algorithm="rendering a graph">
-	1. Prepare the rendering.
-
-		1. Let <var>Q<sub>rendering</sub></var> be an empty [=control message
-			queue=]. [=Atomically=] [=swap=]
-			<var>Q<sub>rendering</sub></var> and <var>Q</var>.
+<div id="rendering-initialization" algorithm="initialize rendering loop">
+	The following step MUST be performance once before the rendering loop starts.
 
-		2. Set the internal slot <dfn attribute
-			for="BaseAudioContext">[[current frame]]</dfn> of the
-			{{BaseAudioContext}} to 0.
+	1. Set the internal slot <dfn attribute
+		for="BaseAudioContext">[[current frame]]</dfn> of the
+		{{BaseAudioContext}} to 0. Also set {{BaseAudioContext/currentTime}} to 0.
```

Comment:
Initialize currentTime as well, otherwise it's undefined until the first render happens.

Reply:
This is done. Or did you want something different?